2996_reflection/reflection.md

```diff
@@ -1534,28 +1529,82 @@ In our initial proposal a value of type `std::meta::info` can represent:
   - any function or member function
   - any variable, static data member, or structured binding
   - any non-static data member
-  - any constant value
+  - any enumerator
   - any template
-  - any namespace
+  - any namespace (including the global namespace) or namespace alias
+  - any object that is a _permitted result of a constant expression_
+  - any value with _structural type_ (with the caveat that any references or pointers must designate objects that are permitted results of constant expressions)
```

Comment:
Use the same term:
```suggestion
- any value with _structural type_ that is a permitted result of a constant expression
```

Actually why do we need the requirement on structural? If we end up passing this value into `substitute` then we'll pick up that requirement on substitution, but like... what's wrong with:
```cpp
class C {
    int i; // private
public:
    constexpr int get() const { return i; }
};

constexpr auto r = meta::reflect_result(C{});
static_assert([: r :].get() == 0);
```

Reply:
Whatever class of things we allow here, the compiler will have to know how to compare. I think we were trying to rule out custom comparison operators, but I don't think "structural" does that for us anyway.

@daveedvdv Any thoughts on what the right class is here? Aggregate < Structural < Literal, AFAICT

Reply:
(applied the suggested change)